fix(jans-casa): bouncy castle errors on startup

This started to appear:

```
jans-casa - 21-04 21:13:32.219 INFO  jans.util.security.SecurityProviderUtility SecurityProviderUtility.java:281- Adding Bouncy Castle Provider
jans-casa - 21-04 21:13:32.219 ERROR jans.util.security.SecurityProviderUtility SecurityProviderUtility.java:267- Security provider 'BC' doesn't exists in class path. Please deploy correct war for this environment!
jans-casa - 21-04 21:13:32.219 ERROR jans.util.security.SecurityProviderUtility SecurityProviderUtility.java:269- org.bouncycastle.jce.provider.BouncyCastleProvider
java.lang.ClassNotFoundException: org.bouncycastle.jce.provider.BouncyCastleProvider
        at java.base/java.net.URLClassLoader.findClass(Unknown Source) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        at org.eclipse.jetty.webapp.WebAppClassLoader.loadClass(WebAppClassLoader.java:511) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
        at java.base/java.lang.Class.forName(Unknown Source) ~[?:?]
        at io.jans.util.security.SecurityProviderUtility.installBCProvider(SecurityProviderUtility.java:284) ~[jans-core-util-1.1.2-SNAPSHOT.jar:?]
        at io.jans.util.security.SecurityProviderUtility.installBCProvider(SecurityProviderUtility.java:265) [jans-core-util-1.1.2-SNAPSHOT.jar:?]
        at io.jans.util.security.SecurityProviderUtility.installBCProvider(SecurityProviderUtility.java:274) [jans-core-util-1.1.2-SNAPSHOT.jar:?]
        at io.jans.casa.core.ConfigurationHandler.inited(ConfigurationHandler.java:76) [classes/:?]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at org.jboss.weld.injection.producer.DefaultLifecycleCallbackInvoker.invokeMethods(DefaultLifecycleCallbackInvoker.java:83) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.injection.producer.DefaultLifecycleCallbackInvoker.postConstruct(DefaultLifecycleCallbackInvoker.java:66) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.injection.producer.BasicInjectionTarget.postConstruct(BasicInjectionTarget.java:122) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.bean.ManagedBean.create(ManagedBean.java:174) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.contexts.AbstractContext.get(AbstractContext.java:96) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.bean.ContextualInstanceStrategy$DefaultContextualInstanceStrategy.get(ContextualInstanceStrategy.java:100) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.bean.ContextualInstanceStrategy$ApplicationScopedContextualInstanceStrategy.get(ContextualInstanceStrategy.java:140) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
at org.jboss.weld.bean.ContextualInstance.get(ContextualInstance.java:50) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.bean.proxy.ContextBeanInstance.getInstance(ContextBeanInstance.java:102) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at org.jboss.weld.bean.proxy.ProxyMethodHandler.invoke(ProxyMethodHandler.java:105) [weld-core-impl-4.0.3.Final.jar:4.0.3.Final]
        at io.jans.casa.core.ConfigurationHandler$Proxy$_$$_WeldClientProxy.init(Unknown Source) [classes/:?]
        at io.jans.casa.core.ZKService.init(ZKService.java:66) [classes/:?]
        at io.jans.casa.core.ZKService$Proxy$_$$_WeldClientProxy.init(Unknown Source) [classes/:?]
        at io.jans.casa.core.init.ZKInitializer.init(ZKInitializer.java:14) [classes/:?]
        at org.zkoss.zk.ui.util.Configuration.invokeWebAppInits(Configuration.java:781) [zk-9.6.0.2-jakarta.jar:9.6.0.2-migrated-1.0.0]
        at org.zkoss.zk.ui.impl.AbstractWebApp.init(AbstractWebApp.java:214) [zk-9.6.0.2-jakarta.jar:9.6.0.2-migrated-1.0.0]
        at org.zkoss.zk.ui.http.SimpleWebApp.init(SimpleWebApp.java:58) [zk-9.6.0.2-jakarta.jar:9.6.0.2-migrated-1.0.0]
        at org.zkoss.zk.ui.http.WebManager.<init>(WebManager.java:250) [zk-9.6.0.2-jakarta.jar:9.6.0.2-migrated-1.0.0]
        at org.zkoss.zk.ui.http.WebManager.<init>(WebManager.java:111) [zk-9.6.0.2-jakarta.jar:9.6.0.2-migrated-1.0.0]
        at org.zkoss.zk.ui.http.HttpSessionListener23.contextInitialized(HttpSessionListener23.java:141) [zk-9.6.0.2-jakarta.jar:9.6.0.2-migrated-1.0.0]
        at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:1046) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:624) [jetty-servlet-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.ContextHandler.contextInitialized(ContextHandler.java:983) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.servlet.ServletHandler.initialize(ServletHandler.java:740) [jetty-servlet-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:392) [jetty-servlet-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1305) [jetty-webapp-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:902) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:306) [jetty-servlet-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:533) [jetty-webapp-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:171) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:121) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:89) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:171) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:121) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:89) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:171) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:89) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.StatisticsHandler.doStart(StatisticsHandler.java:253) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:171) [jetty-util-11.0.20.jar:11.0.20]
at org.eclipse.jetty.server.Server.start(Server.java:470) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:89) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.server.Server.doStart(Server.java:415) [jetty-server-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93) [jetty-util-11.0.20.jar:11.0.20]
        at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1919) [jetty-xml-11.0.20.jar:11.0.20]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at org.eclipse.jetty.start.Main.invokeMain(Main.java:229) [start.jar:11.0.20]
        at org.eclipse.jetty.start.Main.start(Main.java:528) [start.jar:11.0.20]
        at org.eclipse.jetty.start.Main.main(Main.java:76) [start.jar:11.0.20]
```

in email otp plugin:

```
jans-casa - 21-04 21:13:43.309 INFO  org.pf4j.AbstractPluginManager AbstractPluginManager.java:407- Start plugin 'email_2fa_core@1.1.2-SNAPSHOT'
jans-casa - 21-04 21:13:43.316 ERROR org.pf4j.DefaultPluginFactory DefaultPluginFactory.java:69- null
java.lang.reflect.InvocationTargetException: null
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:?]
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:?]
        at org.pf4j.DefaultPluginFactory.createInstance(DefaultPluginFactory.java:65) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.DefaultPluginFactory.create(DefaultPluginFactory.java:59) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.PluginWrapper.getPlugin(PluginWrapper.java:81) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.AbstractPluginManager.startPlugin(AbstractPluginManager.java:408) [pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.core.ExtensionsManager.startPlugin(ExtensionsManager.java:327) [classes/:?]
        at io.jans.casa.core.ExtensionsManager.updatePlugins(ExtensionsManager.java:181) [classes/:?]
        at io.jans.casa.core.ExtensionsManager$Proxy$_$$_WeldClientProxy.updatePlugins(Unknown Source) [classes/:?]
        at io.jans.casa.timer.FSPluginChecker.jobToBeExecuted(FSPluginChecker.java:129) [classes/:?]
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1927) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.notifyListenersBeginning(JobRunShell.java:324) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.run(JobRunShell.java:173) [quartz-2.3.2.jar:?]
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:?]
Caused by: java.lang.NoClassDefFoundError: org/bouncycastle/asn1/ASN1Encodable
        at io.jans.casa.plugins.emailotp.EmailOtpPlugin.<init>(EmailOtpPlugin.java:17) ~[?:?]
        ... 17 more
Caused by: java.lang.ClassNotFoundException: org.bouncycastle.asn1.ASN1Encodable
        at org.pf4j.PluginClassLoader.loadClass(PluginClassLoader.java:144) ~[pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.plugins.emailotp.EmailOtpPlugin.<init>(EmailOtpPlugin.java:17) ~[?:?]
        ... 17 more
jans-casa - 21-04 21:13:43.318 ERROR jans.casa.core.ExtensionsManager ExtensionsManager.java:344- Cannot invoke "org.pf4j.Plugin.start()" because the return value of "org.pf4j.PluginWrapper.getPlugin()" is null
java.lang.NullPointerException: Cannot invoke "org.pf4j.Plugin.start()" because the return value of "org.pf4j.PluginWrapper.getPlugin()" is null
        at org.pf4j.AbstractPluginManager.startPlugin(AbstractPluginManager.java:408) ~[pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.core.ExtensionsManager.startPlugin(ExtensionsManager.java:327) [classes/:?]
        at io.jans.casa.core.ExtensionsManager.updatePlugins(ExtensionsManager.java:181) [classes/:?]
        at io.jans.casa.core.ExtensionsManager$Proxy$_$$_WeldClientProxy.updatePlugins(Unknown Source) [classes/:?]
        at io.jans.casa.timer.FSPluginChecker.jobToBeExecuted(FSPluginChecker.java:129) [classes/:?]
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1927) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.notifyListenersBeginning(JobRunShell.java:324) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.run(JobRunShell.java:173) [quartz-2.3.2.jar:?]
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:?]
jans-casa - 21-04 21:13:43.318 WARN  jans.casa.core.ExtensionsManager ExtensionsManager.java:192- Plugin email_2fa_core failed to start, will be deleted
jans-casa - 21-04 21:13:43.318 INFO  org.pf4j.AbstractPluginManager AbstractPluginManager.java:477- Stop plugin 'email_2fa_core@1.1.2-SNAPSHOT'
jans-casa - 21-04 21:13:43.318 ERROR org.pf4j.DefaultPluginFactory DefaultPluginFactory.java:69- null
java.lang.reflect.InvocationTargetException: null
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:?]
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:?]
        at org.pf4j.DefaultPluginFactory.createInstance(DefaultPluginFactory.java:65) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.DefaultPluginFactory.create(DefaultPluginFactory.java:59) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.PluginWrapper.getPlugin(PluginWrapper.java:81) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.AbstractPluginManager.stopPlugin(AbstractPluginManager.java:478) [pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.AbstractPluginManager.stopPlugin(AbstractPluginManager.java:448) [pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.core.plugin.CasaPluginManager.stopPlugin(CasaPluginManager.java:54) [classes/:?]
        at io.jans.casa.core.ExtensionsManager.stopPlugin(ExtensionsManager.java:285) [classes/:?]
        at io.jans.casa.core.ExtensionsManager.deletePlugin(ExtensionsManager.java:315) [classes/:?]
        at io.jans.casa.core.ExtensionsManager.updatePlugins(ExtensionsManager.java:193) [classes/:?]
        at io.jans.casa.core.ExtensionsManager$Proxy$_$$_WeldClientProxy.updatePlugins(Unknown Source) [classes/:?]
        at io.jans.casa.timer.FSPluginChecker.jobToBeExecuted(FSPluginChecker.java:129) [classes/:?]
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1927) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.notifyListenersBeginning(JobRunShell.java:324) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.run(JobRunShell.java:173) [quartz-2.3.2.jar:?]
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:?]
Caused by: java.lang.NoClassDefFoundError: org/bouncycastle/asn1/ASN1Encodable
        at io.jans.casa.plugins.emailotp.EmailOtpPlugin.<init>(EmailOtpPlugin.java:17) ~[?:?]
        ... 20 more
Caused by: java.lang.ClassNotFoundException: org.bouncycastle.asn1.ASN1Encodable
        at org.pf4j.PluginClassLoader.loadClass(PluginClassLoader.java:144) ~[pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.plugins.emailotp.EmailOtpPlugin.<init>(EmailOtpPlugin.java:17) ~[?:?]
        ... 20 more
jans-casa - 21-04 21:13:43.320 ERROR jans.casa.core.ExtensionsManager ExtensionsManager.java:317- Cannot invoke "org.pf4j.Plugin.stop()" because the return value of "org.pf4j.PluginWrapper.getPlugin()" is null
java.lang.NullPointerException: Cannot invoke "org.pf4j.Plugin.stop()" because the return value of "org.pf4j.PluginWrapper.getPlugin()" is null
        at org.pf4j.AbstractPluginManager.stopPlugin(AbstractPluginManager.java:478) ~[pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.AbstractPluginManager.stopPlugin(AbstractPluginManager.java:448) ~[pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.core.plugin.CasaPluginManager.stopPlugin(CasaPluginManager.java:54) ~[classes/:?]
        at io.jans.casa.core.ExtensionsManager.stopPlugin(ExtensionsManager.java:285) ~[classes/:?]
        at io.jans.casa.core.ExtensionsManager.deletePlugin(ExtensionsManager.java:315) [classes/:?]
        at io.jans.casa.core.ExtensionsManager.updatePlugins(ExtensionsManager.java:193) [classes/:?]
        at io.jans.casa.core.ExtensionsManager$Proxy$_$$_WeldClientProxy.updatePlugins(Unknown Source) [classes/:?]
        at io.jans.casa.timer.FSPluginChecker.jobToBeExecuted(FSPluginChecker.java:129) [classes/:?]
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1927) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.notifyListenersBeginning(JobRunShell.java:324) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.run(JobRunShell.java:173) [quartz-2.3.2.jar:?]
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:?]
jans-casa - 21-04 21:13:43.320 ERROR org.quartz.core.ErrorLogger QuartzScheduler.java:2407- Unable to notify JobListener(s) of Job to be executed: (Job will NOT be executed!). trigger= TimerService.trigger_FSPluginChecker_fschecker job= TimerService.FSPluginChecker_fschecker
org.quartz.SchedulerException: JobListener 'FSPluginChecker_fschecker' threw exception: Index: 5, Size: 0
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1929) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.notifyListenersBeginning(JobRunShell.java:324) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.run(JobRunShell.java:173) [quartz-2.3.2.jar:?]
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:?]
Caused by: java.lang.IndexOutOfBoundsException: Index: 5, Size: 0
        at java.base/java.util.ArrayList.rangeCheckForAdd(Unknown Source) ~[?:?]
        at java.base/java.util.ArrayList.add(Unknown Source) ~[?:?]
        at io.jans.casa.core.ExtensionsManager.updatePlugins(ExtensionsManager.java:194) ~[classes/:?]
        at io.jans.casa.core.ExtensionsManager$Proxy$_$$_WeldClientProxy.updatePlugins(Unknown Source) ~[classes/:?]
        at io.jans.casa.timer.FSPluginChecker.jobToBeExecuted(FSPluginChecker.java:129) ~[classes/:?]
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1927) ~[quartz-2.3.2.jar:?]
        ... 3 more
jans-casa - 21-04 21:14:43.190 INFO  jans.casa.core.ExtensionsManager ExtensionsManager.java:141- Loading new plugins...
jans-casa - 21-04 21:14:43.191 ERROR org.quartz.core.ErrorLogger QuartzScheduler.java:2407- Unable to notify JobListener(s) of Job to be executed: (Job will NOT be executed!). trigger= TimerService.trigger_FSPluginChecker_fschecker job= TimerService.FSPluginChecker_fschecker
org.quartz.SchedulerException: JobListener 'FSPluginChecker_fschecker' threw exception: Plugin '/opt/jans/jetty/jans-casa/plugins/strong-authn-settings-1.1.2-SNAPSHOT.jar' already loaded with id 'strong-authn-settings'
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1929) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.notifyListenersBeginning(JobRunShell.java:324) [quartz-2.3.2.jar:?]
        at org.quartz.core.JobRunShell.run(JobRunShell.java:173) [quartz-2.3.2.jar:?]
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:?]
Caused by: org.pf4j.PluginAlreadyLoadedException: Plugin '/opt/jans/jetty/jans-casa/plugins/strong-authn-settings-1.1.2-SNAPSHOT.jar' already loaded with id 'strong-authn-settings'
        at org.pf4j.AbstractPluginManager.loadPluginFromPath(AbstractPluginManager.java:822) ~[pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.DefaultPluginManager.loadPluginFromPath(DefaultPluginManager.java:139) ~[pf4j-3.10.0.jar:3.10.0]
        at org.pf4j.AbstractPluginManager.loadPlugin(AbstractPluginManager.java:201) ~[pf4j-3.10.0.jar:3.10.0]
        at io.jans.casa.core.plugin.CasaPluginManager.loadPlugin(CasaPluginManager.java:39) ~[classes/:?]
        at io.jans.casa.core.ExtensionsManager.loadPlugin(ExtensionsManager.java:280) ~[classes/:?]
        at io.jans.casa.core.ExtensionsManager.updatePlugins(ExtensionsManager.java:147) ~[classes/:?]
        at io.jans.casa.core.ExtensionsManager$Proxy$_$$_WeldClientProxy.updatePlugins(Unknown Source) ~[classes/:?]
        at io.jans.casa.timer.FSPluginChecker.jobToBeExecuted(FSPluginChecker.java:129) ~[classes/:?]
        at org.quartz.core.QuartzScheduler.notifyJobListenersToBeExecuted(QuartzScheduler.java:1927) ~[quartz-2.3.2.jar:?]
        ... 3 more
```

--- a/jans-casa/plugins/email_2fa_core/pom.xml
+++ b/jans-casa/plugins/email_2fa_core/pom.xml
@@ -12,6 +12,7 @@
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>11</maven.compiler.target>
 		<plugin.id>email_2fa_core</plugin.id>
+        <bc.version>1.78</bc.version>
 	</properties>
 
 	<repositories>
@@ -87,25 +88,25 @@
 		<dependency>
 			<groupId>org.bouncycastle</groupId>
 			<artifactId>bcprov-jdk18on</artifactId>
-			<version>1.74</version>
+			<version>${bc.version}</version>
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.bouncycastle</groupId>
-			<artifactId>bcprov-jdk18on</artifactId>
-			<version>1.74</version>
+			<artifactId>bcprov-jdk18on</artifactId> 
+			<version>${bc.version}</version>
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.bouncycastle</groupId>
 			<artifactId>bcpkix-jdk18on</artifactId>
-			<version>1.73</version>
+			<version>${bc.version}</version>
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.bouncycastle</groupId>
 			<artifactId>bcmail-jdk18on</artifactId>
-			<version>1.73</version>
+			<version>${bc.version}</version>
 			<scope>provided</scope>
 		</dependency>
 	</dependencies>

--- a/jans-casa/pom.xml
+++ b/jans-casa/pom.xml
@@ -25,7 +25,7 @@
         <ldapsdk.version>6.0.8</ldapsdk.version>
         <resteasy.version>6.0.3.Final</resteasy.version>
         <tika.version>2.6.0</tika.version>
-        <bc.version>1.74</bc.version>
+        <bc.version>1.78</bc.version>
         <!-- SonarCloud integration properties -->
         <sonar.projectKey>JanssenProject_jans-casa</sonar.projectKey>
         <sonar.moduleKey>${project.groupId}:${project.artifactId}</sonar.moduleKey>

